solutions/bronze/usaco-833.mdx

```diff
@@ -781,3 +769,95 @@ with open("family.out", "w") as written:
 
 </PySection>
 </LanguageSection>
+
+
+## Solution 3
```

Comment:
added a solution that doesn't explicitly find the lowest common ancestor (imo this is the most natural way to approach the problem ...)